vassal-app/src/main/java/VASSAL/build/module/map/boardPicker/board/mapgrid/RegularGridNumbering.java

```diff
@@ -273,36 +274,54 @@ public void addPropertyChangeListener(PropertyChangeListener l) {
 
   @Override
   public String[] getAttributeDescriptions() {
-    return new String[]{"Order:  ",
-                        "Separator:  ",
-                        "Horizontal numbering:  ",
-                        "Leading zeros in horizontal:  ",
-                        "Starting number in horizontal:  ",
-                        "Horizontal numbering descending?",
-                        "Vertical numbering:  ",
-                        "Leading zeros in vertical:  ",
-                        "Starting number in vertical:  ",
-                        "Vertical numbering descending?",
-                        "Location format:  ",
-                        "Draw Numbering?",
-                        "Font size:  ",
-                        "Color:  ",
-                        "Rotate text (Degrees):  ",
-                        "Text X offset:  ",
-                        "Text Y offset:  "};
+    return new String[]{
+      Resources.getString("Editor.RegularGridNumbering.order"),
+      Resources.getString("Editor.RegularGridNumbering.separator"),
+      Resources.getString("Editor.RegularGridNumbering.horizontal_numbering"),
+      Resources.getString("Editor.RegularGridNumbering.leading_zeros_in_horizontal"),
+      Resources.getString("Editor.RegularGridNumbering.starting_number_in_horizontal"),
+      Resources.getString("Editor.RegularGridNumbering.horizontal_numbering_descending"),
+      Resources.getString("Editor.RegularGridNumbering.vertical_numbering"),
+      Resources.getString("Editor.RegularGridNumbering.leading_zeros_in_vertical"),
+      Resources.getString("Editor.RegularGridNumbering.starting_number_in_vertical"),
+      Resources.getString("Editor.RegularGridNumbering.vertical_numbering_descending"),
+      Resources.getString("Editor.RegularGridNumbering.location_format"),
+      Resources.getString("Editor.RegularGridNumbering.draw_numbering"),
+      Resources.getString("Editor.RegularGridNumbering.font_size"),
+      Resources.getString("Editor.RegularGridNumbering.color"),
```

Comment:
Wherever possible, use the generic Editor key at the top of the Editor.properties, rather than creating a second key for the same text.
Editor.font_size
Editor.color